CMake/boost/FindBoost.cmake

```diff
@@ -0,0 +1 @@
+set(Boost_FOUND TRUE)
```

Comment:
nit: Why are these CMake/*/Find* files required ?

Reply:
To override the modules that come with cmake. As they look for already built libs at configure time (aka when we haven't built the dependencies yet) they won't find them.

Cmake >= 3.24 comes with a built in way to do this (`OVERRIDE_FIND_PACKAGE`) which I tested and works great but that is out of reach for now :D